sql/src/main/java/io/crate/analyze/UnboundAnalyzer.java

```diff
@@ -161,7 +161,10 @@ public AnalyzedStatement visitShowCreateTable(ShowCreateTable node, Analysis con
 
         @Override
         public AnalyzedStatement visitShowSessionParameter(ShowSessionParameter node, Analysis context) {
-            return new ShowSessionParameterAnalyzedStatement(node.parameter());
+            Query query = showStatementAnalyzer.rewriteShowSessionParameter(node);
```

Comment:
why do we not validate the setting name here but on boundAnalysis only?

Reply:
will add the validation here as well.